TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/test_cli.py

```diff
@@ -635,15 +665,17 @@ def test_cli_metadata_when_not_defined(runner, source_cluster_only_yaml_path):
 
 
 def test_cli_metadata_migrate(runner, mocker):
-    mock = mocker.patch("subprocess.run")
+    mock_subprocess_result = SimpleNamespace(stdout="Command successful", stderr=None)
```

Comment:
I think I'm missing something here, but what's a `SimpleNamespace`?

Reply:
In this context I'm using it as a Mock object that I can provide any fields to. I think I can make this better by creating the actual object the subprocess returns instead of this shortcut but let me check